phpstan.neon

```diff
@@ -0,0 +1,14 @@
+parameters:
+    level: 8
+    paths:
+        - src
+    bootstrapFiles:
+        - tests/bootstrap.php
+    checkGenericClassInNonGenericObjectType: false
```

Comment:
This is deprecated and should be removed. Please fix possible error messages ....
```suggestion
```